modules/mod_comment/mod_comment.erl

```diff
@@ -52,8 +53,9 @@ event(#submit{message={newcomment, Args}, form=FormId}, Context) ->
             Email = <<"">>
     end,
     Message = z_context:get_q_validated(<<"message">>, Context),
+    Rating = get_rating(Context),
     Is_visible = case m_config:get_value(comments, moderate, Context) of <<"1">> -> false; _Else -> true end,
```

Comment:
Line 57 is too long:     Is_visible = case m_config:get_value(comments, moderate, Context) of <<"1">> -> false; _Else -> true end,.